test/cli/cli-codegen-javascript.spec.ts

```diff
@@ -115,20 +115,22 @@ it('should print load/save storageState', async ({ runCLI, testInfo }) => {
   const saveFileName = testInfo.outputPath('save.json');
   await fs.promises.writeFile(loadFileName, JSON.stringify({ cookies: [], origins: [] }), 'utf8');
   const cli = runCLI([`--load-storage=${loadFileName}`, `--save-storage=${saveFileName}`, 'codegen', emptyHTML]);
-  const expectedResult = `const { chromium } = require('playwright');
+  const expectedResult1 = `const { chromium } = require('playwright');
 
 (async () => {
   const browser = await chromium.launch({
     headless: false
   });
   const context = await browser.newContext({
     storageState: '${loadFileName}'
-  });
+  });`;
+  await cli.waitFor(expectedResult1);
```

Comment:
Perhaps we should accumulate results in the cli fixture, so that we can wait for a large chunk as before?